editor/editor_help.cpp

```diff
@@ -3735,6 +3735,7 @@ EditorHelpBit::EditorHelpBit(const String &p_symbol) {
 
 	title = memnew(RichTextLabel);
 	title->set_theme_type_variation("EditorHelpBitTitle");
+	title->set_custom_minimum_size(Size2(512 * EDSCALE, 0)); // GH-93031. Set the minimum width since `fit_content` is true.
```

Comment:
```suggestion
	title->set_custom_minimum_size(Size2(512 * EDSCALE, 0)); // GH-93031. Set the minimum width even if `fit_content` is true.
```

Reply:
This is a suggestion to `self`, do you want to make that change or get feedback on whether it would be better?

Both seem fine to me.

Reply:
The comment was misleading. I just wanted to explain the probable reason why I made this mistake. The documentation says `If true, the label's minimum size will be automatically updated to fit its content, matching the behavior of Label.`, but this only applies to the height, not the width.

The minimum width here is needed in both cases, regardless of `fit_content`.